File: explorer-backend/src/Models/API/BlockchainInfo.cs
using ExplorerBackend.Models.Node.Response;

namespace ExplorerBackend.Models.API;

public class BlockchainInfo
{
    public int CurrentSyncedBlock { get; set; }
    public GetBlockchainInfoResult? ChainInfo { get; set; }
    public GetChainalgoStatsResult? AlgoStats { get; set; }
    public Dictionary<string, double>? NetworkHashrates { get; set; }
}

File: explorer-frontend/src/models/API/BlockchainInfo.ts
import type { GetBlockchainInfoResult } from "@/models/Node/GetBlockchainInfoResult";
import type { GetChainalgoStatsResult } from "@/models/Node/GetChainalgoStatsResult";

export interface BlockchainInfo {
  currentSyncedBlock: number;
  chainInfo: GetBlockchainInfoResult;
  algoStats: GetChainalgoStatsResult;
}